Dockerfile.csi-sanity

```diff
@@ -68,4 +70,4 @@ RUN /csi-sanity -csi.version &&\
 # other csi-sanity options: --ginkgo.v -ginkgo.noColor
 #TODO remove "--ginkgo.skip" option after fixing volume paginatin by token:
 ENTRYPOINT ["/bin/sh", "-c", "/run-driver && /csi-sanity --csi.endpoint=${SOCK} --csi.createpathcmdtimeout=180 --csi.removepathcmdtimeout=180 --ginkgo.skip 'pagination should detect volumes added between pages and accept tokens when the last volume from a page is deleted'"]
-#ENTRYPOINT ["/bin/sh", "-c", "/run-driver && /csi-sanity --csi.endpoint=${SOCK} --csi.createpathcmdtimeout=180 --csi.removepathcmdtimeout=180]
\ No newline at end of file
+#ENTRYPOINT ["/bin/sh", "-c", "/run-driver && /csi-sanity --csi.endpoint=${SOCK} --csi.createpathcmdtimeout=180 --csi.removepathcmdtimeout=180]
```

Comment:
minor: Is no newline at end of the file is deliberately removed?

Reply:
Yes, I am trying to have a newline at the end of all files and this one was missing it.